avaframe/in3Utils/fileHandlerUtils.py

```diff
@@ -164,20 +167,28 @@ def exportcom1DFAOutput(avaDir, cfg=''):
 
     # Export peak files and reports
     for k in range(sNo):
-        shutil.copy('%s%.03f/%s/raster/%s_pfd.asc' % (resPath, logDict[varPar][k], logDict['simName'][k],
-                    logDict['simName'][k]),
-                    '%s/%s_%.03f_pfd.asc' % (outDirPF, logDict['simName'][k], logDict[varPar][k]))
-        shutil.copy('%s%.03f/%s/raster/%s_ppr.asc' % (resPath, logDict[varPar][k], logDict['simName'][k],
-                    logDict['simName'][k]),
-                    '%s/%s_%.03f_ppr.asc' % (outDirPF, logDict['simName'][k], logDict[varPar][k]))
-        shutil.copy('%s%.03f/%s/raster/%s_pv.asc' % (resPath, logDict[varPar][k], logDict['simName'][k],
-                    logDict['simName'][k]),
-                    '%s/%s_%.03f_pv.asc' % (outDirPF, logDict['simName'][k], logDict[varPar][k]))
-        shutil.copy('%s%.03f/%s.html' % (resPath, logDict[varPar][k], logDict['simName'][k]),
-                    '%s/%s_%.03f.html' % (outDirRep, logDict['simName'][k], logDict[varPar][k]))
+        pathFrom = os.path.join('%s%.03f' % (resPath, logDict[varPar][k]),
+                                '%s' % logDict['simName'][k], 'raster',
```

Comment:
here for example would't it be enough to write: 
os.path.join('%s%s.03f' %(resPath, logDict[varPar][k]), logDict['simName'][k], .....
so if it is one part of the path without a separator just use the variable instead of doing '%s' % variable?

Reply:
this is the case at several locations